frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -68,41 +71,58 @@ export class ReduxTlmDispatcher extends React.Component {
     ])
   }
 
+  handleNotification (data) {
+    if (this.props.user.userId !== data.fields.author.user_id) {
+      this.props.dispatch(addNotification(data))
+    }
+  }
+
   handleWorkspaceDeleted = data => {
-    this.props.dispatch(removeWorkspace(data.fields.workspace))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(removeWorkspace(data.fields.workspace))
+    this.handleNotification(data)
   }
 
   handleWorkspaceModified = data => {
-    this.props.dispatch(updateWorkspaceDetail(data.fields.workspace))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(updateWorkspaceDetail(data.fields.workspace))
+    this.handleNotification(data)
   }
 
   handleMemberCreated = data => {
-    this.props.dispatch(addWorkspaceMember(data.fields.user, data.fields.workspace.workspace_id, data.fields.member))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(addWorkspaceMember(data.fields.user, data.fields.workspace.workspace_id, data.fields.member))
+    this.handleNotification(data)
   }
 
   handleMemberModified = data => {
-    this.props.dispatch(updateWorkspaceMember(data.fields.user, data.fields.workspace.workspace_id, data.fields.member))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(updateWorkspaceMember(data.fields.user, data.fields.workspace.workspace_id, data.fields.member))
+    this.handleNotification(data)
   }
 
   handleMemberDeleted = data => {
-    this.props.dispatch(removeWorkspaceMember(data.fields.user.user_id, data.fields.workspace.workspace_id))
-    if (this.props.user.userId === data.fields.user.user_id) this.props.dispatch(removeWorkspace(data.fields.workspace))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(removeWorkspaceMember(data.fields.user.user_id, data.fields.workspace.workspace_id))
+    if (props.user.userId === data.fields.user.user_id) props.dispatch(removeWorkspace(data.fields.workspace))
+    this.handleNotification(data)
+  }
+
+  handleMentionCreated = data => {
+    this.handleNotification(data)
   }
 
   handleContentCreated = data => {
-    this.props.dispatch(addWorkspaceContentList([data.fields.content], data.fields.workspace.workspace_id))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(addWorkspaceContentList([data.fields.content], data.fields.workspace.workspace_id))
+    this.handleNotification(data)
   }
 
   handleContentCommentCreated = async data => {
-    if (data.fields.author.user_id === this.props.user.userId) return
+    const { props } = this
+    if (data.fields.author.user_id === props.user.userId) return
```

Comment:
Same comment as above, shouldn't the condition be `the tlm comes from this tab/window`? That we can test with `data.fields.clientTokenId`?

Reply:
Ping @gbonaspetti

Reply:
Use the clientTokenId will make the changes inside this if only for this tab, but we want to set the read state for all tabs that the user is connected, right?

Reply:
Yes, you're right.